src/chrome/content/rules/Bit.ly_vanity_domains.xml

```diff
@@ -1,4 +1,4 @@
-<ruleset name="bit.ly vanity domains">
```

Comment:
I think this is probably over-broad. There are a lot of domains in this file. Which ones were caught by your method?